scripts/RESTORE.md

```diff
@@ -0,0 +1,109 @@
+# Backup and Restore Guide
+
+## Backup Process
+
+### Creating a Backup
+
+1. **Ensure services are running** - The backup process requires the database to be accessible
+2. **Run the backup script**:
+   ```bash
+   # Use default backup directory (data/backups/)
+   ./scripts/backup.sh
+   
+   # Use custom backup directory
+   ./scripts/backup.sh /path/to/backups
+   
+   # Show help
+   ./scripts/backup.sh -h
+   ```
+3. **Backup location** - Backups are stored in the specified directory (default: `data/backups/`)
+4. **Backup naming** - Files are named with timestamp: `shutter-api-keyper-YYYY-MM-DDTHH-MM-SS.tar.xz`
+
+### What Gets Backed Up
+
+- Database dump (`keyper.dump`) - Contains full schema and data from the `keyper` database
+- Chain data (`data/chain/`) - Blockchain data and configuration
+- Keyper configuration (`config/`) - Application configuration files
+- Environment variables - Except Signing Key
+
+### What's NOT Backed Up
+
+- **Signing Key** - The `SIGNING_KEY` environment variable is intentionally excluded from backups for security reasons. You must manually preserve this value separately.
+
+### Security Considerations
+
+⚠️ **IMPORTANT**: Backup files contain sensitive information including:
+- Database contents with potentially sensitive data
+- Configuration files that may contain API keys, passwords, or other secrets
+- Chain data that could be used to reconstruct transaction history
+
+**Security Best Practices:**
+- Store backups on a different machine or secure cloud storage
+- Limit access to backup files to authorized personnel only
+- Consider using backup encryption tools for additional security
+
+## Restore Process
+
+### Prerequisites
+
+- **Empty keyper instance** - The restore *must* be performed on a fresh, empty deployment
+- **No running services** - Ensure all Docker containers are stopped before restore
+- **Backup file available** - The backup archive should be present in the specified backup directory
+- **Signing key available** - You must have the original `SIGNING_KEY` value from your deployment
+
+### Restore Steps
+
+1. **Setup environment**:
+   ```bash
+   cp example-api.env .env
+   # Edit the .env file to insert your `SIGNING_KEY`, the rest of the settings will be restored from the backup.
+   ```
```

Comment:
Move this to the end

```suggestion
1. **Setup environment**:
   ```bash
   cp example-api.env .env
   # Edit the .env file to insert your `SIGNING_KEY`, the rest of the settings will have been restored from the backup.
   ```
```